### PR TITLE
Update information-schema slow query & Statement.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-slow-query.md [hunk 1]
```diff
@@ -7,6 +7,14 @@ summary: Learn the `SLOW_QUERY` information_schema table.
 
 The `SLOW_QUERY` table provides the slow query information of the current node, which is the parsing result of the TiDB slow log file. The column names in the table are corresponding to the field names in the slow log.
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the `Slow Query Table` is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (ran-huang) [inline information-schema/information-schema-slow-query.md:14]:
```suggestion
> Currently, the `SLOW_QUERY` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```

File: statement-summary-tables.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ Therefore, starting from v4.0.0-rc.1, TiDB provides system tables in `informatio
 - [`cluster_statements_summary_history`](#statements_summary_evicted)
 - [`statements_summary_evicted`](#statements_summary_evicted)
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the `Statement Summary Table` is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (ran-huang) [inline statement-summary-tables.md:22]:
```suggestion
> Currently, the `statements_summary` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```

File: information-schema/information-schema-slow-query.md [hunk 3]
```diff
@@ -7,6 +7,14 @@ summary: Learn the `SLOW_QUERY` information_schema table.
```
Comment (ran-huang) [inline information-schema/information-schema-slow-query.md:7]: Can you add these limitations in https://docs.pingcap.com/tidbcloud/serverless-tier-limitations

File: information-schema/information-schema-slow-query.md (hunk 1 above)
Comment (qiancai) [reply information-schema/information-schema-slow-query.md:14]: If we don't plan to support this table in near future for Serverless Tier, we could remove "Currently".

File: statement-summary-tables.md [hunk 4]
```diff
@@ -15,6 +15,14 @@ Therefore, starting from v4.0.0-rc.1, TiDB provides system tables in `informatio
 - [`cluster_statements_summary_history`](#statements_summary_evicted)
 - [`statements_summary_evicted`](#statements_summary_evicted)
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the `statements_summary` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (ran-huang) [inline statement-summary-tables.md:22]:
```suggestion
> Currently, the following tables are unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta):  `statements_summary`, `statements_summary_history`, `cluster_statements_summary`, and `cluster_statements_summary_history`.
```

File: information-schema/information-schema-slow-query.md [hunk 5]
```diff
@@ -7,6 +7,14 @@ summary: Learn the `SLOW_QUERY` information_schema table.
 
 The `SLOW_QUERY` table provides the slow query information of the current node, which is the parsing result of the TiDB slow log file. The column names in the table are corresponding to the field names in the slow log.
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the `SLOW_QUERY` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```
Comment (qiancai) [inline information-schema/information-schema-slow-query.md:14]: If we do not plan to provide the support of these table for Serverless Tiers in near future, can we remove "Currently" from this note?

Comment (lilyjazz) [reply information-schema/information-schema-slow-query.md:14]: OK.

Comment (lilyjazz) [inline information-schema/information-schema-slow-query.md:14]:
```suggestion
> The `SLOW_QUERY` table is unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta).
```

File: statement-summary-tables.md [hunk 6]
```diff
@@ -15,6 +15,14 @@ Therefore, starting from v4.0.0-rc.1, TiDB provides system tables in `informatio
 - [`cluster_statements_summary_history`](#statements_summary_evicted)
 - [`statements_summary_evicted`](#statements_summary_evicted)
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> Currently, the following tables are unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta):  `statements_summary`, `statements_summary_history`, `cluster_statements_summary`, and `cluster_statements_summary_history`.
```
Comment (lilyjazz) [inline statement-summary-tables.md:22]:
```suggestion
> The following tables are unavailable for [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta):  `statements_summary`, `statements_summary_history`, `cluster_statements_summary`, and `cluster_statements_summary_history`.
```

File: tidb-cloud/serverless-tier-limitations.md [hunk 7]
```diff
@@ -24,6 +24,10 @@ We are constantly filling in the feature gaps between Serverless Tier and Dedica
 - [Time to live (TTL)](/time-to-live.md) is not available for Serverless Tier clusters currently.
 - The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) syntax is not applicable to TiDB Cloud [Serverless Tier](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) clusters.
 
+## System Tables
+
+- Tables `CLUSTER_SLOW_QUERY`, `SLOW_QUERY`, `CLUSTER_STATEMENTS_SUMMARY`, `CLUSTER_STATEMENTS_SUMMARY_HISTORY`, `STATEMENTS_SUMMARY`, `STATEMENTS_SUMMARY_HISTORY` are not available for Serverless Tier clusters currently.
```
Comment (lilyjazz) [inline tidb-cloud/serverless-tier-limitations.md:29]:
```suggestion
- Tables `CLUSTER_SLOW_QUERY`, `SLOW_QUERY`, `CLUSTER_STATEMENTS_SUMMARY`, `CLUSTER_STATEMENTS_SUMMARY_HISTORY`, `STATEMENTS_SUMMARY`, `STATEMENTS_SUMMARY_HISTORY` are not available for Serverless Tier clusters.
```

**Review by ran-huang — APPROVED** [review]

**Comment by qiancai** [comment]
other LGTM

**Pull request merged**